### DISCUSSION TITLE
Thin mode for database version 11

### DISCUSSION THREAD

**Discussion by keha94** [oracle/python-oracledb#96] [Q&A] [answered]
Will thin mode become available for Oracle database version 11g in the future?

Or are there technical limitations that prevent this from happening?

**Comment by cjbj** [comment, marked as answer]
Support for 11g is not on our roadmap - technically it's not trivial. More importantly Oracle's paid support for DB 11gR2 itself finishes soon, so you should (ideally!) be upgrading your databases.  We have other features to add to Thin mode (and Thick mode) that we are focusing on.
